src/hooks/useRouteAssetId/useRouteAssetId.ts

```diff
@@ -29,7 +29,12 @@ const getRouteAssetId = (pathname: string) => {
     chainNamespace?: ChainNamespace
     chainReference?: ChainReference
   }>(pathname, {
-    path: ['/accounts/:accountId/:assetId', '/accounts/:chainNamespace\\::chainReference\\:(.+)'],
+    path: [
```

Comment:
See the path above `/lending/pool${path}` goes first and since the AssetId match runs first, `/lending/pool/:accountId/:assetId` as it and be rugged.
Made it `poolAccount` to avoid these shenanigans.